generated/nitclk/__init__.py

```diff
@@ -6,7 +6,18 @@
 
 from nitclk.errors import DriverWarning   # noqa: F401
 from nitclk.errors import Error     # noqa: F401
-from nitclk.session import Session  # noqa: F401
+from nitclk.session import SessionReference  # noqa: F401
+
+# Function imports
+from nitclk.session import configure_for_homogeneous_triggers  # noqa: F401
```

Comment:
> F401 | module imported but unused

Is this needed?

Reply:
Yes. It is imported and not used in this file. `flake8` doesn't care that it is `__init__.py` and is supposed to do things like this.